kura/org.eclipse.kura.web2/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,8 @@ Bundle-Version: 2.0.0.qualifier
 Bundle-Vendor: EUROTECH
```

Comment:
@dwoodard1 I got many class not found exceptions related to missing classes.
I could solve these issues only after adding javax.xml.parsers, org.w3c.dom, org.xml.sax.
Here the final version that worked to me:

Manifest-Version: 1.0
Bundle-ManifestVersion: 2
Bundle-Name: org.eclipse.kura.web2
Bundle-SymbolicName: org.eclipse.kura.web;singleton:=true
Bundle-Version: 2.0.0.qualifier
Bundle-Vendor: EUROTECH
Bundle-RequiredExecutionEnvironment: JavaSE-1.6
Service-Component: OSGI-INF/*.xml
Import-Package: com.google.gwt.http.client,
 com.google.gwt.safehtml.shared,
 com.google.gwt.user.client,
 com.google.gwt.user.client.rpc,
 com.google.gwt.user.client.rpc.core.java.lang,
 com.google.gwt.user.client.rpc.core.java.math,
 com.google.gwt.user.client.rpc.core.java.sql,
 com.google.gwt.user.client.rpc.core.java.util,
 com.google.gwt.user.client.rpc.core.java.util.logging,
 com.google.gwt.user.client.rpc.impl,
 com.google.gwt.user.client.ui,
 com.google.gwt.user.linker.rpc,
 com.google.gwt.user.rebind.rpc,
 com.google.gwt.user.server.rpc,
 com.google.gwt.user.server.rpc.impl,
 javax.crypto,
 javax.crypto.spec,
 javax.security.auth,
 javax.servlet;version="2.6.0",
 javax.servlet.http;version="2.6.0",
 javax.xml.bind;resolution:=optional,
 javax.xml.parsers,
 org.apache.commons.fileupload;version="1.2.2",
 org.apache.commons.fileupload.disk;version="1.2.2",
 org.apache.commons.fileupload.servlet;version="1.2.2",
 org.apache.commons.io;version="1.4.0",
 org.apache.commons.net.util;version="3.1.0",
 org.eclipse.kura;version="[1.0,2.0)",
 org.eclipse.kura.certificate;version="[1.0,2.0)",
 org.eclipse.kura.command;version="[1.1,2.0)",
 org.eclipse.kura.configuration;version="[1.0,2.0)",
 org.eclipse.kura.configuration.metatype;version="[1.0,2.0)",
 org.eclipse.kura.core.configuration;version="[1.0,2.0)",
 org.eclipse.kura.core.configuration.metatype;version="[1.0,2.0)",
 org.eclipse.kura.core.configuration.util;version="[1.0,2.0)",
 org.eclipse.kura.core.net.util;version="[1.0,2.0)",
 org.eclipse.kura.core.util;version="[1.0,2.0)",
 org.eclipse.kura.crypto;version="[1.0,2.0)",
 org.eclipse.kura.data;version="[1.0,2.0)",
 org.eclipse.kura.db;version="[1.0,2.0)",
 org.eclipse.kura.deployment.agent;version="[1.0,2.0)",
 org.eclipse.kura.net;version="[1.0,2.0)",
 org.eclipse.kura.net.dhcp;version="[1.0,2.0)",
 org.eclipse.kura.net.firewall;version="[1.0,2.0)",
 org.eclipse.kura.net.modem;version="[1.0,2.0)",
 org.eclipse.kura.net.wifi;version="[1.0,2.0)",
 org.eclipse.kura.position;version="[1.0,2.0)",
 org.eclipse.kura.security;version="[1.0,2.0)",
 org.eclipse.kura.ssl;version="[1.0,2.0)",
 org.eclipse.kura.system;version="[1.0,2.0)",
 org.eclipse.kura.usb;version="[1.0,2.0)",
 org.osgi.framework,
 org.osgi.service.component;version="1.2.0",
 org.osgi.service.deploymentadmin,
 org.osgi.service.event,
 org.osgi.service.http,
 org.osgi.service.metatype;version="1.2.0",
 org.osgi.util.measurement;version="1.0.1",
 org.osgi.util.position;version="1.0.1",
 org.slf4j;version="1.6.4",
 org.w3c.dom,
 org.xml.sax
Bundle-ClassPath: .